services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/IntentionalRerouteSpec.groovy

```diff
@@ -180,6 +181,7 @@ class IntentionalRerouteSpec extends BaseSpecification {
         flowHelper.deleteFlow(flow.id)
     }
 
+    @Tags(SMOKE)
```

Comment:
Rare case. All cases with `ignoreBandwidth=true` should be considered rare. It is more important to properly reroute regular, "normal" flows

Reply:
removed